rotation.py

```diff
@@ -22,38 +22,31 @@ def __str__(self) -> str:
     def __repr__(self) -> str:
         return str(self)
 
-    @staticmethod
-    def are_commuting(a: 'PauliOperator', b: 'PauliOperator') -> bool:
-        """
-        Check if 2 PauliOperators commute or anti-commute.
-
-        Returns:
-            bool: True if they commute, False if they anti-commute
-        """
-        return True if PauliOperator.I in (a,b) or a == b else False
 
     @staticmethod
-    def multiply_by_i(a: 'PauliOperator', b: 'PauliOperator') -> 'PauliOperator':
+    def are_commuting(a: 'PauliOperator', b: 'PauliOperator') -> Tuple[bool, 'PauliOperator']:
         """
-        Return iAB.
+        Check if 2 PauliOperators A and Bcommute or anti-commute. 
+        Returns True and AB if commute or False and iAB if anti-commute.
+
```

Comment:
Might be worth saying that the second returned value is C=iAB when they anticommute and C=AB when they commute

Reply:
will update the description later